koap/build.gradle.kts

```
@@ -1,9 +1,13 @@
+import lt.petuska.npm.publish.dsl.NpmAccess
```

ultra nitpick

```suggestion
import lt.petuska.npm.publish.dsl.NpmAccess.RESTRICTED
```

Then below, `NpmAccess.RESTRICTED` can be replaced with `RESTRICTED`.